graylog2-web-interface/src/components/users/UserDetails/PreferencesSection.jsx

```diff
@@ -0,0 +1,52 @@
+// @flow strict
+import * as React from 'react';
+
+import { PREFERENCES_THEME_MODE } from 'theme/constants';
+import Store from 'logic/local-storage/Store';
+import User from 'logic/users/User';
+import ReadOnlyFormGroup from 'components/common/ReadOnlyFormGroup';
+import SectionComponent from 'components/common/Section/SectionComponent';
+
+type Props = {
+  user: User,
+};
+
+export const getPreferenceValueLabel = (preferenceType: string, preferenceValue: mixed): string => {
+  if (preferenceType === 'boolean') {
+    return preferenceValue ? 'yes' : 'no';
+  }
+
+  return String(preferenceValue);
+};
```

Comment:
I moved the `getPreferenceValueLabel` logic to the `ReadOnlyFormGroup`, instead of a `valueConverter` prop I used `typeof` and made sure the provided values are a boolean.

I like suggestion regarding the the visual presentation.
![image](https://user-images.githubusercontent.com/46300478/91065970-645fb000-e631-11ea-9968-e98f9dc25e3e.png)

Reply:
I thought about increasing the icon size, but in my opinion it looks better this way.